packages/e2e/package.json

```diff
@@ -149,6 +149,7 @@
     "babel-loader": "^8.2.5",
     "blake2b-no-wasm": "2.1.4",
     "buffer": "^6.0.3",
+    "chromedriver": "^120.0.0",
```

Comment:
This dependency was removed in the past because the actual version has to be in sync with the developer environment chrome version, which can vary.
Ideally devs would have the desired chromedriver installed globally in their setup, or use an alternative solution.
If we choose to keep the version here, [this README comment](https://github.com/input-output-hk/cardano-js-sdk/blob/master/README.md#L77) should be updated to inform the need to set this version based on the developer's chrome browser version.
Sidenote: the instructions in the [README](https://github.com/input-output-hk/cardano-js-sdk/blob/master/README.md#L77) do not work 😞

Reply:
It was re-added because global installation approach no longer works. I think we should remove the instructions and try to come up with a better way to manage this in the future

Reply:
Agree. Let's replace the instructions with "Update chromedrive dependency to the local chrome version", or something in the lines of that :D

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/aba05830935a2d5a441802fab53ccbcd5aea4a32